[2023-10-07 12:12:52,574] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:11:37.339054+00:00 [queued]>
[2023-10-07 12:12:52,581] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:11:37.339054+00:00 [queued]>
[2023-10-07 12:12:52,582] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:12:52,582] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-07 12:12:52,583] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:12:52,591] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_d_customer> on 2023-10-07 12:11:37.339054+00:00
[2023-10-07 12:12:52,594] {standard_task_runner.py:52} INFO - Started process 3819 to run task
[2023-10-07 12:12:52,596] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'update_d_customer', 'manual__2023-10-07T12:11:37.339054+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/sprint3.py', '--cfg-path', '/tmp/tmpwtu_9n4a', '--error-file', '/tmp/tmpo6msyilg']
[2023-10-07 12:12:52,597] {standard_task_runner.py:80} INFO - Job 87: Subtask update_d_customer
[2023-10-07 12:12:52,633] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:11:37.339054+00:00 [running]> on host 70ec72ace0d7
[2023-10-07 12:12:52,672] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=update_d_customer
AIRFLOW_CTX_EXECUTION_DATE=2023-10-07T12:11:37.339054+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-07T12:11:37.339054+00:00
[2023-10-07 12:12:52,678] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2023-10-07 12:12:52,682] {sql.py:315} INFO - Running statement: insert into mart.d_customer (customer_id, first_name, last_name, city_id)
select customer_id, first_name, last_name, max(city_id) from staging.user_order_log
where customer_id not in (select customer_id from mart.d_customer)
group by customer_id, first_name, last_name, parameters: None
[2023-10-07 12:12:52,692] {sql.py:324} INFO - Rows affected: 1087
[2023-10-07 12:12:52,703] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart, task_id=update_d_customer, execution_date=20231007T121137, start_date=20231007T121252, end_date=20231007T121252
[2023-10-07 12:12:52,729] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-07 12:12:52,753] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
